src/test/java/AcmeShop.java

```diff
@@ -114,10 +107,105 @@ public void addingToCart() {
             builder4.moveToElement(fieldComments, width, height).clickAndHold().moveByOffset(0, height*5)
                     .release().build();
             builder4.perform();
```

Comment:
no need for this, add .perform() to the initial chain.
The reason it didn't work for you was that the exception was thrown because you were trying to drag out of bounds of the current browser window.